files/pt-br/glossary/css_selector/index.md

```diff
@@ -50,4 +55,4 @@ Nessa primeira regra do CSS, eu estou selecionando o elemento da div estilizando
 
 ### Referência técnica
 
-{{SpecName("CSS3 Selectors")}}
+[Selectors Level 3](https://drafts.csswg.org/selectors-3/)
```

Comment:
If the macro call is not in a specification table, just replace it with the corresponding url. For the rest of content, follow the rule